examples/timer-log-spring/README.adoc

```diff
@@ -0,0 +1,58 @@
+= timer-log
+
+This is a basic hello world example that uses Spring to set-up
+a Camel timer that triggers every second and prints to the
+log.
+
+TIP: Check the https://camel.apache.org/camel-quarkus/latest/first-steps.html[Camel Quarkus User guide] for prerequisites
+and other general information.
+
+== Start in the Development mode
+
+[source,shell]
+----
+$ mvn clean compile quarkus:dev -DnoDeps
+----
+
+The above command compiles the project, starts the application and lets the Quarkus tooling watch for changes in your
+workspace. Any modifications in your project will automatically take effect in the running application.
+
+TIP: Please refer to the Development mode section of
+https://camel.apache.org/camel-quarkus/latest/first-steps.html#_development_mode[Camel Quarkus User guide] for more details.
+
+Then look at the log output in the console. As we run the example
+in Quarkus Dev Mode, you can edit the source code and have live updates.
+For example try to change the logging output to be `Bye World` or the property `timer.period` in application.properties.
```

Comment:
The `Bye World` suggestion is a bit outdated. The line is now `.setBody(() -> "Incremented the counter: " + counter.increment())` so we could perhaps suggest 

```
For example try to change `Incremented the counter` to `Counter state` in the message body or change the property `timer.period` in application.properties.

The same holds for the CDI example.